rdrf/rdrf/patients/query_data.py

```diff
@@ -1,10 +1,10 @@
 import logging
 
 from gql_query_builder import GqlQuery
+from graphql import GraphQLError
```

Comment:
Better to raise our own exception rather than one from a library. We can still catch `GraphQLError`s as well.

Reply:
Do you foresee any issues if I just raise a general `Exception` rather than defining a type for it?

I've updated my `except` block in `form_view` to catch any `Exception` instead of just `GraphQLError`.

Reply:
I think if there was some other error like a database issue or something, we wouldn't want to catch that. The exception should just bubble up to django and it can return a 500 response.

Reply:
I went with a hybrid approach and created a new one using GraphQLError as the base.
I can change this though if you would prefer no link between them.